From: Shengjing Zhu <zhsj@debian.org>
Date: Sun, 27 Nov 2022 03:46:48 +0800
Subject: Use github.com/golang/protobuf/ptypes/empty

google/protobuf/empty.proto in libprotobuf-dev (3.21.9) has switched to
google.golang.org/protobuf/types/known/emptypb
---
 trustmanager/remoteks/generator.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trustmanager/remoteks/generator.go b/trustmanager/remoteks/generator.go
index 93f0b8b..ae48ad5 100644
--- a/trustmanager/remoteks/generator.go
+++ b/trustmanager/remoteks/generator.go
@@ -2,4 +2,4 @@ package remoteks
 
 // this file exists solely to allow us to use `go generate` to build our
 // compiled GRPC interface for Go.
-//go:generate protoc -I ./ ./keystore.proto --go_out=plugins=grpc:.
+//go:generate protoc -I ./ ./keystore.proto --go_out=plugins=grpc,Mgoogle/protobuf/empty.proto=github.com/golang/protobuf/ptypes/empty:.
